src/index.html

```diff
@@ -6,9 +6,75 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+        alt="logo"
+      >
+        <img src="./images/logo.png" alt="moyo_logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__element">
+            <a href="#" class="is-active nav__element__link">
```

Comment:
It's generally recommended to avoid nesting subelements in BEM (e.g `block__element__element`). Calling it `nav__link` works just fine :)
```suggestion
            <a href="#" class="is-active nav__link">
```